### PR TITLE
html: parsing: log warning if two input node classes share same type

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: lona/html/parsing.py [hunk 1]
```diff
@@ -49,14 +50,24 @@ def _find_node_classes(module):
 
                     continue
 
+                if node_class.ATTRIBUTES['type'] in INPUT_NODE_CLASSES:
+                    logger.warning(
+                        'WARNING: Two input Node classes with the same type=%s were found: %r and %r',  # NOQA: E501
```
Comment (fscherf) [inline lona/html/parsing.py:55]:
```suggestion
'WARNING: Two input Node classes with type "%s" were found: %r and %r',  # NOQA: E501
```

File: lona/html/parsing.py [hunk 2]
```diff
@@ -49,14 +50,24 @@ def _find_node_classes(module):
...
+                    )
+
                 INPUT_NODE_CLASSES[node_class.ATTRIBUTES['type']] = node_class
 
             # nodes
             else:
                 if node_class.TAG_NAME in NODE_CLASSES:
                     logger.warning(
-                        'WARNING: Two Node classes with the same tag name were found: %s',  # NOQA: E501
+                        'WARNING: Two Node classes with the same tag name %s were found: %r and %r',  # NOQA: E501
```
Comment (fscherf) [inline lona/html/parsing.py:67]:
```suggestion
'WARNING: Two Node classes with the tag name "%s" were found: %r and %r',  # NOQA: E501
```

**Review by fscherf — CHANGES_REQUESTED** [review]
@maratori Works great! I think the messages are easier to read with quoting

File: lona/html/parsing.py (hunk 1 above)
Comment (maratori) [reply lona/html/parsing.py]: Fixed

File: lona/html/parsing.py (hunk 2 above)
Comment (maratori) [reply lona/html/parsing.py]: Fixed

**Comment by fscherf** [comment]
@maratori Great! Thanks! :)

**Pull request merged**